examples/cluster-e2e/monitoring.tf

```diff
@@ -0,0 +1,28 @@
+#####################################################
+# SYSDIG Configuration
+# Copyright 2021 IBM
+#####################################################
+
+module "sysdig_instance" {
+  source = "terraform-ibm-modules/observability/ibm//modules/monitoring-sysdig"
```

Comment:
same as above

Reply:
To provision a sysdig instance we need to source to registry level